src/lib/eina/eina_thread.c

```diff
@@ -212,8 +106,8 @@ eina_thread_name_set(Eina_Thread t, const char *name)
      }
    else buf[0] = 0;
 #ifndef __linux__
-   pthread_set_name_np((pthread_t)t, buf);
-   return EINA_TRUE;
+   
+   return _eina_thread_set_name_win32(t, buf);
 #else
    if (pthread_setname_np((pthread_t)t, buf) == 0) return EINA_TRUE;
```

Comment:
Who does this work without #include <pthread.h>?

Reply:
Now, there is nothing about pthread_* function in eina_thread.c.